### ISSUE TITLE
Add compile time option to build without 64-bit support

### ISSUE THREAD

**Issue by GoogleCodeExporter** [nanopb/nanopb#86]
```
Currently nanopb requires the compiler to support uint64_t. It might be worth 
it to have a option like PB_NO_64BIT to disable the parts that require 
uint64_t, in order to support more limited compilers.
```

Original issue reported on code.google.com by `Petteri.Aimonen` on 11 Sep 2013 at 1:49

**Comment by PetteriAimonen** [comment]
Haven't had anyone actually request this in 4 years, so closing off for now.

**Comment by vervaekejonathan** [comment]
I want this!

**Comment by PetteriAimonen** [comment]
Ok :)

**Comment by wcappelle** [comment]
I'm also interested in this option (for both size optimizations & speed penalty)

**Comment by vervaekejonathan** [comment]
Related to this issue, shifting over more than 32bits is not available on all platforms - I would be nice if this was linked with the 'without 64bit'-option

**Comment by smorin-primero** [comment]
We were looking at using protobuffer for our limited platform with an old compiler (which we have no way to upgrade unfortunately) and it does not have 64 bit integer support. An option like this would be a must for us too.

**Comment by PetteriAimonen** [comment]
Should be implemented now.. testing is very welcome, as these platforms might be limited in more ways than I've predicted.

**Comment by vervaekejonathan** [comment]
excellent news, will test it on an 8-bit platform

**Comment by PetteriAimonen** [comment]
Fix released in nanopb-0.3.9.
